### DISCUSSION TITLE
Error with TIL page deployment

### DISCUSSION THREAD

**Discussion by 60eokk** [community/community#59869] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have been working on my github TIL to keep track of what Im learning and write a lot of things down.
But recently Ive been getting errors when I commit. 
It doesnt happen every time, and I do not do anything particular. All I do is just write things down in the posts and commit. But why am I getting this error?

<img width="763" alt="Screenshot 2023-07-01 at 10 01 11 PM" src="https://github.com/community/community/assets/117603244/41fa6e26-10b7-46ff-a5c1-0c9b71191d12">

**Comment by JamesMGreene** [comment, marked as answer]
**TL;DR:** You aren't doing anything wrong! You just so happen to be pushing a new commit before the previous one has finished deploying.

You've run into a concurrency problem that we (GitHub) need to resolve. Apologies for that! 😓 

If you're interested, I've written a bit more about it in this open issue for the underlying GitHub Action we are using to trigger the Pages deployment:
- https://github.com/actions/deploy-pages/issues/118#issuecomment-1478298088
